pkg/intelmq-packaging/adduser.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ -z "$HOST_UID" -o -z "$HOST_USERNAME" ]; then
+if [ -z "$HOST_UID" ] || [ "$HOST_USERNAME" ]; then
```

Comment:
I do like the original version better, as it is only one test expression instread of two. (YMMV)
However, if you wan't to keep the change please note that it changed the semantics (introducing a bug) as the -z is now missing in the second test.

Reply:
shellcheck said here `[ p -o q ] is not well defined`. https://www.shellcheck.net/wiki/SC2166 also says the expression is obsolete. ([also](https://github.com/koalaman/shellcheck/issues/489)

thanks for finding the loss of the second `-z`